zoia_lib/backend/api.py

```diff
@@ -127,6 +130,373 @@ def download(self, idx: str):
             # No patch with the supplied id was found.
             return None
 
+    def _get_license_data(self):
```

Comment:
It might be worthwhile to extract this (an the categories data) out to its own JSON file to load. If nothing else this would help reduce the size of this file which may make future changes more managable.